DirectProgramming/DPC++FPGA/Tutorials/Features/read_only_cache/README.md

```diff
@@ -0,0 +1,261 @@
+# Read-Only Cache 
+This FPGA tutorial demonstrates how to use the read-only cache feature to boost
+the throughput of an FPGA DPC++ design that uses a read-only look-up table
+defined in host code.
+
+***Documentation***:  The [DPC++ FPGA Code Samples
+Guide](https://software.intel.com/content/www/us/en/develop/articles/explore-dpcpp-through-intel-fpga-code-samples.html)
+helps you to navigate the samples and build your knowledge of DPC++ for FPGA.
+<br>
+The [oneAPI DPC++ FPGA Optimization
+Guide](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide)
+is the reference manual for targeting FPGAs through DPC++. <br>
+The [oneAPI Programming
+Guide](https://software.intel.com/en-us/oneapi-programming-guide) is a general
+resource for target-independent DPC++ programming.
+
+| Optimized for                     | Description
+---                                 |---
+| OS                                | Linux* Ubuntu* 18.04/20.04, RHEL*/CentOS* 8, SUSE* 15; Windows* 10
+| Hardware                          | Intel® Programmable Acceleration Card (PAC) with Intel Arria® 10 GX FPGA <br> Intel® FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix® 10 SX) <br> Intel® FPGA 3rd party / custom platforms with oneAPI support <br> *__Note__: Intel® FPGA PAC hardware is only compatible with Ubuntu 18.04* 
+| Software                          | Intel® oneAPI DPC++ Compiler <br> Intel® FPGA Add-On for oneAPI Base Toolkit 
+| What you will learn               | How and when to use the read-only cache feature
+| Time to complete                  | 30 minutes
+
+
+
+## Purpose
+
+This FPGA tutorial demonstrates an example of using the read-only cache to
+boost the throughput of an FPGA DPC++ design. Specifically, the read-only cache
+is most appropriate for table lookups that are constant throughout the
+execution of a kernel and is optimized for high cache hit performance.
```

Comment:
This is a bit unclear: is the read-only cache optimized for high cache hit performance, or are the table lookups optimized for high cache hit performance?  (actually, I guess the conjugation of 'is' instead of 'are' gives the answer, but that's a bit subtle - I think this could be made more clear)